frontend/src/components/Histogram.jsx

```diff
@@ -83,6 +105,26 @@ export default class Histogram extends React.Component {
                 dataKey="weekly_avg_counterfactual"
                 stroke="#ff7300"
               />
+              if (this.state.first_restrictions_date != null){" "}
```

Comment:
Can this just be `if (this.state.first_restrictions_date)`? I know that would work in some languages but not sure about Javascript.